packages/app/src/app/pages/Sandbox/Editor/Content/index.js

```diff
@@ -185,15 +185,16 @@ class EditorPreview extends React.Component<Props, State> {
     );
     const disposeModuleHandler = reaction(
       () => [store.editor.currentModule, store.editor.currentModule.code],
-      () => {
+      ([newModule, _]) => {
```

Comment:
I think you can just change this line to:

`([newModule]) => {`

Reply:
True, thanks for your input.